frontend/src/store/reducer.js

```diff
@@ -1,27 +1,28 @@
 const initialState = {
-    offsetPosts: 0,
-    jwt: null,
-    reload: 0
+  offsetPosts: 0,
```

Comment:
(I'm surer and surer: your Prettier doesn't check `.js` files... Please ask someone for their conf, or ask it from me if needed)